src/hii_popdens.py

```diff
@@ -31,9 +42,23 @@ def calc(self):
         gpw_taskdate = gpw_prior.add(gpw_diff_fraction)
         gpw_taskdate_300m = gpw_taskdate.resample().reproject(crs=self.crs, scale=self.scale)
 
+
+        caspian = ee.FeatureCollection(self.inputs['caspian']['ee_path'])
+
+        jrc = ee.Image(self.inputs['jrc']['ee_path'])\
+                        .select('occurrence')\
+                        .lte(75)\
+                        .unmask(1)\
+                        .multiply(ee.Image(0).clip(caspian).unmask(1))
+
+        ocean = ee.Image(self.inputs['ocean']['ee_path'])
+
+
         gpw_venter = gpw_taskdate_300m.add(ee.Image(1))\
             .log()\
-            .multiply(ee.Image(3.333))
+            .multiply(ee.Image(3.333))\
+            .updateMask(jrc)\
+            .updateMask(ocean)
```

Comment:
Seems to me we should just have one "non-terrestrial" mask precalculated once -- i.e. could we not combine surface water, ocean, and Caspian into one mask?